src/StackExchange.Redis/RedisDatabase.cs

```diff
@@ -4504,17 +4535,17 @@ private Message GetStringGetWithExpiryMessage(RedisKey key, CommandFlags flags,
                 long seconds = milliseconds / 1000;
                 return when switch
                 {
-                    When.Always    => Message.Create(Database, flags, RedisCommand.SET, key, value, RedisLiterals.EX, seconds, RedisLiterals.GET),
-                    When.Exists    => Message.Create(Database, flags, RedisCommand.SET, key, value, RedisLiterals.EX, seconds, RedisLiterals.XX, RedisLiterals.GET),
+                    When.Always => Message.Create(Database, flags, RedisCommand.SET, key, value, RedisLiterals.EX, seconds, RedisLiterals.GET),
+                    When.Exists => Message.Create(Database, flags, RedisCommand.SET, key, value, RedisLiterals.EX, seconds, RedisLiterals.XX, RedisLiterals.GET),
                     When.NotExists => Message.Create(Database, flags, RedisCommand.SET, key, value, RedisLiterals.EX, seconds, RedisLiterals.NX, RedisLiterals.GET),
                     _ => throw new ArgumentOutOfRangeException(nameof(when)),
                 };
             }
 
             return when switch
             {
-                When.Always    => Message.Create(Database, flags, RedisCommand.SET, key, value, RedisLiterals.PX, milliseconds, RedisLiterals.GET),
-                When.Exists    => Message.Create(Database, flags, RedisCommand.SET, key, value, RedisLiterals.PX, milliseconds, RedisLiterals.XX, RedisLiterals.GET),
+                When.Always => Message.Create(Database, flags, RedisCommand.SET, key, value, RedisLiterals.PX, milliseconds, RedisLiterals.GET),
+                When.Exists => Message.Create(Database, flags, RedisCommand.SET, key, value, RedisLiterals.PX, milliseconds, RedisLiterals.XX, RedisLiterals.GET),
```

Comment:
There's a lot of whitespace changes in here which affect git history - can we please remove the non-diff so we only have relevant changes in the PR?

Reply:
removed those changes. 
here i see tendency to keep it more readable with manual formatting/aligning and i appreciate the effort . Most of the whitespace diff comes from that. But here it seems we kinda leave the auto-format features in IDE, which is a great thing,  and there are other formatting styles we see like braces with/without new line, improper white spaces before/after keywords etc... which causes codebase has non-standard formatting and usages here and there. 
do you think it is a fair trade off ? 

to be able to use auto-fomat, i can send another dedicated PR just to fix these whitespace diff show up here,  how that sounds ??

Reply:
Sorry for the delay here - your notes are totally valid, going to add StyleCop and do one big cleanup pass to enforce consistency with more PRs coming in, but want to get your existing efforts merged first so I eat the diff rather than you on cleanup.

Reply:
then we are good to merge this, right ?